wandb/sdk/wandb_artifacts.py

```diff
@@ -859,8 +859,13 @@ def name(cls) -> str:
     def from_config(cls, config: Dict) -> "WandbStoragePolicy":
         return cls(config=config)
 
-    def __init__(self, config: Optional[Dict] = None) -> None:
-        self._cache = get_artifacts_cache()
+    def __init__(
+        self,
+        config: Optional[Dict] = None,
+        cache: Optional[ArtifactsCache] = None,
+        api: Optional[InternalApi] = None,
+    ) -> None:
+        self._cache = cache if cache is not None else get_artifacts_cache()
```

Comment:
nit?
```suggestion
        self._cache = cache or get_artifacts_cache()
```

Reply:
Seems fine! I just do `x if x is not None else y` by default because `x or y` can be a bug if `x`'s type can be falsey. But that's not likely to happen for `ArtifactCache`, I guess.